### PR TITLE
Adding 3 modules to content

### PR DESCRIPTION
**Pull request by mcgonagle**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/050_module_1/51_overview.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Setting up your AWS account
+
+{{% children %}}
+
+{{% notice warning %}}
+You are responsible for the cost of the AWS services used while running this workshop in your AWS account. We highly recommend you to go to the [request AWS credit page](/030_self_guided_setup/30_request_credit.html) so you can run this workshop without any charge to you.
+{{% /notice %}}
+
+1. If you don't already have an AWS account with Administrator access: [create
+one now by clicking here](https://aws.amazon.com/getting-started/)
```
Comment (mariswa) [inline content/050_module_1/51_overview.md:16]: Thomas\Peter, we already have separate section for AWS account prep etc in "Self Guided Setup" 

I would add any IAM configuration needed there. Also images are not loading - looks like you didn't bring it. What IAM role is for? Can we give YAML for the policy instead as well?

File: content/050_module_1/54_create_application.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: "2. Create the Application"
+chapter: true
+weight: 12
```
Comment (mariswa) [inline content/050_module_1/54_create_application.md:4]: I would add Create the application and the Load Balancer to either setup Armory or separate "Setup test application" (this is what hello-today is, am I correct?) This module called "Create Your First Pipeline" already

File: content/050_module_1/58_add_manual_judgement_stage_and_test_deployment.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+title: "6. Configure workshop specific requirements"
```
Comment (mariswa) [inline content/050_module_1/58_add_manual_judgement_stage_and_test_deployment.md:2]: "Configure workshop specific requirements" title doesn't match "Expand the pipeline: add manual judgment and test deployment" content - should the title be "Expand the pipeline" or "Test deployment"?

File: content/050_module_1/58_add_manual_judgement_stage_and_test_deployment.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    spec:
+      containers:
+        - image: 'justinrlee/nginx:${parameters["tag"]}'
+          name: primary
+          ports:
+            - containerPort: 80
+  </code>
+</pre>
+
+5. Click "Save Changes"
```
Comment (mariswa) [inline content/050_module_1/58_add_manual_judgement_stage_and_test_deployment.md:72]: The numbering is broken here

File: content/050_module_1/51_overview.md (hunk 1 above)
Comment (eugenemu) [reply content/050_module_1/51_overview.md:16]: This file isn't necessary.

File: content/060_module_2/61_edit_your_pipeline.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+            spec:
+              containers:
+              - image: 'justinrlee/nginx:${parameters["tag"]}'
+                name: primary
+                ports:
+                - containerPort: 80
+                  protocol: TCP
+        ```
+
+    1. Below the manifest block, go to the "Rollout Strategy Options"
```
Comment (eugenemu) [inline content/060_module_2/61_edit_your_pipeline.md:58]: Let's add more context here: what other rollout strategy options are offered. Why are we doing it this way, what are the pros and cons of different rollout strategies. What are the use cases behind each etc.

File: content/050_module_1/51_overview.md (hunk 1 above)
Comment (mariswa) [reply content/050_module_1/51_overview.md:16]: No, but better to have prerequisites & setup separately. We can easily adopt the workshop than for any run (self-pace, DevDay etc). At this point looks like its actual hands-on on Armory functionality

**Pull request merged**